### PR TITLE
refactor(card): consolidate variables(#986)

### LINKED ISSUE #986: Use a configuration first method (modifiers and states) for Card
Relates directly back to #273

### PR DESCRIPTION
**Pull request by christiemolloy**
This issue closes #986 

- consolidated padding variables

### CODE REVIEW

**Comment by christiemolloy** [comment]
- I asked CSS Army if we should group property values onto a line or on separate lines 
- I really like the second approach, I think thats a great idea to condense assignments
- Yes to the third point!!

File: src/patternfly/components/Card/card.scss [hunk 1]
```diff
@@ -37,30 +32,21 @@
...
   }
 }
 
 .pf-c-card__header {
   flex: 0 0 auto;
-  padding-top: var(--pf-c-card__header--PaddingTop);
-  padding-right: var(--pf-c-card__header--PaddingRight);
-  padding-bottom: var(--pf-c-card__header--PaddingBottom);
-  padding-left: var(--pf-c-card__header--PaddingLeft);
+  padding: var(--pf-c-card__header--PaddingTop) var(--pf-c-card__header--PaddingX) var(--pf-c-card__header--PaddingBottom) var(--pf-c-card__header--PaddingX);
```
Comment (mcoker) [inline src/patternfly/components/Card/card.scss]:
```suggestion
  padding: var(--pf-c-card__header--PaddingTop) var(--pf-c-card__header--PaddingX) var(--pf-c-card__header--PaddingBottom);
```

**Comment by mcoker** [comment]
Looks great! Just one more comment.

**Comment by christiemolloy** [comment]
Thanks for the review. Made updates

**Pull request merged**